Solo girl in the Dakshinkashi.txt

```diff
@@ -0,0 +1,19 @@
+Hi again, the monsoons have begun and everything around just looks so beautiful. 
+once in a while don't we all wish we could be Naina from Yeh Jawani Hai Deewani movie? Randomly deciding to take a solo trip to the mountains. 
+
+They say if you can travel to places and eat at a restaurant all by yourself without feeling weird about it, that gives you the confidence to take on the world fearlessly!
+Being a Introvert, I would usually wait at the entrance of a cafe until my friends arrived and didn't go in alone and also always felt a little noticed while I had to eat at a restaurant all by myself. But things took a turn when I decided to stay alone in the big buzzing city. 
+Grocery shopping, running other errands and long strolls in the parks, all with myself felt good and realized how dependent it had made me on others. I agree that humans are designed to live together, but living by yourself gives you a sense of freedom you did not realize was needed. 
+
+I have a long list of places to visit and not all our famous touristy spots, I am a lover of infamous small towns and love to watch the daily hustle of the localities. My plans were often being cancelled or postponed due to unavailability at least one friend. That is when I decided. If you want to see a place. You get there! And that is how this this random Solo trip to Temple town of Kalasa happened!
```

Comment:
among lots of famous places to visit in my travel checklist** etc.. I's can be changed to my, mine, me etc sentences